src/Command.js

```diff
@@ -1,19 +1,23 @@
 import _ from "lodash";
 import dedent from "dedent";
+import log from "npmlog";
+
 import ChildProcessUtilities from "./ChildProcessUtilities";
 import FileSystemUtilities from "./FileSystemUtilities";
 import GitUtilities from "./GitUtilities";
-import ExitHandler from "./ExitHandler";
-import progressBar from "./progressBar";
-import Repository from "./Repository";
 import PackageUtilities from "./PackageUtilities";
-import logger from "./logger";
+import Repository from "./Repository";
+import filterFlags from "./utils/filterFlags";
+import writeLogFile from "./utils/writeLogFile";
```

Comment:
I tried to resist it, but I organized all of the imports in every file, first grouped by source (dependency or local), then alphabetized(ish, in this case).